app/services/logger.js

```diff
@@ -0,0 +1,43 @@
+const bunyan = require('bunyan');
+const PrettyStream = require('bunyan-prettystream');
+const { get } = require('lodash');
+
+const Logger = require('./base_logger');
+
+const pkg = require('../../package.json');
+const config = require('../config').get('logger');
+
+const env = process.env.NODE_ENV || 'dev';
+
+const loggerOptions = {
+  name: config.name || pkg.name,
+  streams: [
+    { level: 'error', stream: process.stderr },
+    { level: 'error', path: 'logs/error.log' },
+    ...get(config, 'streams', [])
+  ],
+  serializers: bunyan.stdSerializers,
+};
```

Comment:
Strange construction for logger. Please create correct logger configuration in our config file, for example:
{
"log": {
    "name": "event-emitter",
    "streams": [{
      "level": "debug",
      "path": "logs/event-emitter.log",
       period: '1d',
       count: 3
    }]
  }
}